src/localize.ts

```diff
@@ -13,14 +13,10 @@ export class Localize {
   }
 
   private init() {
-    try {
-      this.options = {
-        ...this.options,
-        ...JSON.parse(process.env.VSCODE_NLS_CONFIG || "{}")
-      };
-    } catch (err) {
-      throw err;
-    }
+    this.options = {
```

Comment:
Any reason to remove exception handling?

Reply:
It doesn't do anything. And this rule enabled in recommended: https://eslint.org/docs/rules/no-useless-catch